src/helpers.ts

```diff
@@ -0,0 +1,23 @@
+import { assert } from 'superstruct';
+
+import { EthAccountType } from './eth';
+import type { InternalAccount } from './internal';
+import { InternalAccountStruct } from './internal';
+
+/**
+ * Checks if the given internal account is of the EVM account type.
+ * @param account - The internal account to check.
+ * @returns Returns true if the internal account has an EVM account type, false otherwise.
+ */
+export function isEVMAccount(account: InternalAccount): boolean {
```

Comment:
Should be named `isEvmAccount`, since we went for `Eoa` instead of `EOA`

(I used this as a reference: https://google.github.io/styleguide/tsguide.html#camel-case)